keymanager-runtime/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "oasis-runtime-keymanager"
+version = "0.3.0-alpha"
+authors = ["Oasis Labs Inc. <info@oasislabs.com>"]
+edition = "2018"
+
+[package.metadata.fortanix-sgx]
+heap-size = 134217728
+stack-size = 2097152
+threads = 2
+
+[dependencies]
+# The referenced commit has rust code compatible with artifacts in oasis-core
```

Comment:
Remove comment

Reply:
done :+1: merging this since CI passed, and only changes after it were removing the comments